### ISSUE TITLE
NIP Checker errors.

### ISSUE THREAD

**Issue by Teffeq** [kolton/d2bot-with-kolbot#3126]
Hi i have recently installed Nip Checker. 
Using Nipchecker at items is working fine (Zero errors) but when i use it on my players file  i get maybe 600 errors as you can see in the picture.
Does NIP Check only work for items or am i doing it wrong?

Would love some help <3![Untitled](https://user-images.githubusercontent.com/89435739/130707649-6676d479-36be-485c-a2dc-3dc3ee6e6cef.png)

**Comment by Samael36** [comment]
Yes, that is normal. As far as I know it only verifies nip files.
If there is an error in your player config, it will show it in game upon starting a new game/script.

**Comment by Teffeq** [comment]
Yeah well that is sad, because for me 99% of all errors I get are from the character file not items...

**Comment by Samael36** [comment]
Can you take a screenshot of one of the errors?
Maybe I can help you locate the line from the character file.
